api/src/org/labkey/api/reader/TabLoader.java

```diff
@@ -70,6 +70,8 @@ public class TabLoader extends DataLoader
     public static final FileType TSV_FILE_TYPE = new TabFileType(Arrays.asList(".tsv", ".txt"), ".tsv", "text/tab-separated-values");
     public static final FileType CSV_FILE_TYPE = new TabFileType(Collections.singletonList(".csv"), ".csv", "text/comma-separated-values");
 
+    private boolean _IncludeComments = false;
```

Comment:
```suggestion
    private boolean _includeComments = false;
```

Casing is weird here.